hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -1014,6 +1014,11 @@ update env ledger st ev = case (st, ev) of
   -- General
   (currentState, OnChainEvent Rollback{rolledBackChainState}) ->
     NewState (setChainState rolledBackChainState currentState) []
+  -- FIXME: We do not write the chain state unless we do a head state change.
```

Comment:
suggestion: move to a github issue? Stuff in code comments have a natural tendency to get lost in the sands of time and trumped by other priorities

Reply:
Was not intending to commit this and I have written a bug report yesterday about this: https://github.com/input-output-hk/hydra/issues/931